spec/context/dimension.fmf

```diff
@@ -12,7 +12,8 @@ description: |
     distro
         The operating system distribution on which the application
         runs (fedora, fedora-33, centos, centos-8, centos-8.4,
-        rhel, rhel-8, rhel-8.4).
+        rhel, rhel-8, rhel-8.4). The "el-8" abbreviation can be
```

Comment:
could it be just `el8` instead?, for some reason, `el-8` looks so weird

Reply:
I don't think so. This would need adjustments on the `fmf` side related to parsing the version which could be quite confusing I guess. For example `mariadb55`. If you think it's important, let's file a new fmf issue and discuss it there.